### PR TITLE
Drop HTTP since its not a necessary explicit dependency

### PR DESCRIPTION
**Pull request by quinnj**
I'm not sure if the FilePathsBase compats are correct, so please correct me if those won't work.

### CODE REVIEW

**Comment by quinnj** [comment]
bors try

**Comment by quinnj** [comment]
bors try

**Comment by quinnj** [comment]
Good catch; bumped.

**Comment by quinnj** [comment]
bors try

File: .github/workflows/CompatHelper.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ jobs:
     steps:
       - uses: julia-actions/setup-julia@latest
         with:
-          version: "1.3"
+          version: "1"
```
Comment (iamed2) [inline .github/workflows/CompatHelper.yml:13]:
```suggestion
          version: "1.6"
```

We don't want CompatHelper suggesting changes that aren't compatible with the minimum Julia version (AFAICT from various GitHub and Slack conversations, the resolution algorithm is currently tied to the running version of Julia)

**Review by ericphanson — APPROVED** [review]
LGTM!

**Pull request merged**